flocker/provision/_libcloud.py

```diff
@@ -217,6 +243,8 @@ class LibcloudProvisioner(object):
         libcloud's ``create_node``.
     :ivar callable provision: Function to call to provision a node.
     :ivar str default_size: Name of the default size of node to create.
+    :ivar str default_user: Function to provide the default username on the
```

Comment:
This `ivar` is not of type `str`.

:question: I think change the name of this variable, it implies that it should be a `str`.

:question: `Function` > `Callable`.